packages/framework/src/Component/Router/FriendlyUrl/FriendlyUrlGenerator.php

```diff
@@ -90,20 +79,7 @@ public function generateFromRouteCollection(
 
         $domainId = $domainConfig->getId();
 
-        if ($this->mainFriendlyUrlSlugCache !== null) {
```

Comment:
I am confused here - why is this block removed, please? From now on, `mainFriendlyUrlSlugCache` should be always defined, so I would expect you to remove the `else` block, not the `if` block, or am I missing something? :thinking:

Reply:
No, you don't miss anything... This is my mistake. Thanks for pointing it out! fixed in 9bcc51f99